packages/ember/src/components/mktg/navbar.gts

```diff
@@ -0,0 +1,66 @@
+import Component from '@glimmer/component';
+import type { ComponentLike } from '@glint/template';
+import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import Button from '../button.gts';
+
+export interface NavbarSignature {
+  Element: HTMLElement;
+  Blocks: {
+    default: [];
+    brand: [];
+    'desktop-action-button': [ComponentLike<Button>];
+    'mobile-action-button': [ComponentLike<Button>];
+  };
+}
+
+export default class NavbarComponent extends Component<NavbarSignature> {
+  @tracked
+  isMenuOpen = false;
+
+  @action
+  toggleMenu() {
+    this.isMenuOpen = !this.isMenuOpen;
+  }
+
+  get classList() {
+    const classes = ['collapse', 'navbar-collapse'];
+    if (this.isMenuOpen) {
+      classes.push('show');
+    }
+    return classes.join(' ');
+  }
+
+  get menuOpen() {
+    return this.isMenuOpen;
+  }
+
+  get menuIcon() {
+    return this.menuOpen ? 'bi-x' : 'bi-list';
+  }
+
+  // TODO: --bs-navbar-toggler-focus-width: 0;
+
+  <template>
+    <nav class="navbar navbar-expand-lg navbar-light bg-primary">
+      {{yield to="brand"}}
+      {{yield (component Button) to="mobile-action-button"}}
```

Comment:
Can this block be combined with the `desktop-action-button` block, and maybe renamed as `actions`?
```suggestion
      {{#if this.responsive.isMobileDevice}}
        {{yield to="actions"}}
      {{/if}}
```